OpenRA.Mods.Common/Widgets/Logic/SettingsLogic.cs

```diff
@@ -452,7 +452,7 @@ Action InitInputPanel(Widget panel)
 				};
 
 				var header = ScrollItemWidget.Setup(hotkeyHeader, returnTrue, doNothing);
-				header.Get<LabelWidget>("LABEL").GetText = () => "Game Commands";
+				header.Get<LabelWidget>("LABEL").GetText = () => "Game commands";
```

Comment:
For consistency - Developer and Music commands were already lower case 'c'. 'Unit stance Commands' looks to me worse so I unified all to 'c'. Should I unify all to 'C' or keep those 2 exceptions?